4337/src/utils/execution.ts

```diff
@@ -91,7 +89,7 @@ export const safeApproveHash = async (
 }
 
 export const safeSignTypedData = async (
-  signer: Signer & TypedDataSigner,
+  signer: Signer,
   safe: Contract,
```

Comment:
We could change the parameter type to the safe address, couldn't we? (Just saying, this may be on the border of the scope of the PR)

Reply:
I changed where possible to use the `safeAddress` as a parameter instead of a full `Safe` contract instance.